icons/cone.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m12 2 9 17c0 1.7-4 3-9 3s-9-1.3-9-3Z" />
```

Comment:
I think this could definitely have a taller base:
```suggestion
  <path d="M12 2 3 18A9 4 0 0 0 21 18Z" />
```

![image](https://github.com/lucide-icons/lucide/assets/17746067/6a975672-63fa-405d-9cfe-1703450bb76f)
![image](https://github.com/lucide-icons/lucide/assets/17746067/e674b8bd-4189-497e-a2e6-dc83c289de4e)

Reply:
@karsa-mistmere That looks better, but shouldn’t it match `cylinder`/`database`…?

Reply:
I prefer @karsa-mistmere proposal.

Reply:
> @karsa-mistmere That looks better, but shouldn’t it match `cylinder`/`database`…?

Ugh, I've completely forgotten about that... 😩 

In general, I guess yes, it should. In this instance however, I think it's better to make this more easily recognisable even if we lose some consistency, since without the added base height, it looks too much like a pie slice.

Side by side they don't look that bad, I think:
![image](https://github.com/lucide-icons/lucide/assets/17746067/757dcf8e-acbf-4d5f-bc81-97bceb6998ee)

Reply:
Hmm, I much prefer the version I already `push`ed 😅 …It’s much more obviously a cone.

This is where we need that poll bot again…